### DISCUSSION TITLE
How to hide points grid in Merkaartor?

### DISCUSSION THREAD

**Discussion by ghost** [openstreetmap/merkaartor#283] [Q&A] [answered]
Is there a way to hide points grid?

![Screenshot_2023-02-28_02-59-39](https://user-images.githubusercontent.com/13188580/221724787-f08011e5-bbe9-424f-bf9c-67863186bcbd.png)

**Comment by ghost** [comment, marked as answer]
UPDATE: Okay, just found out how to hide it:

1. Go to menu "Show";
2. Toggle off "Downloaded areas".

There is no such note in "Documentation", so it might be useful to add it on OSM Wiki:

- https://wiki.openstreetmap.org/wiki/Merkaartor/Documentation

**Reply by Krakonos** [reply]
The wiki needs a bunch of updates. Feel free to help out. I'm not sure where to put it.

I would consider replacing "Downloaded areas" to "Downloaded areas grid" to improve the discoverability, as one would naturally look into "Show" menu, but may not think the grid is a visualisation of "Downloaded areas".

**Reply by ghost** [reply]
> The wiki needs a bunch of updates. Feel free to help out.

I would try to look on it.

> I would consider replacing "Downloaded areas" to "Downloaded areas grid" to improve the discoverability

I'm just curious, is there any reason this option exists, as it it is not moving with canvas and shown with empty window and even there is local OSM file loaded into app. It works only for downloaded OSM regions by Merkaartor, but not for local OSM files?

Also, s it safe for display to keep such static dot pattern for a long period of time during editing?

![Screenshot_2023-02-28_10-10-56](https://user-images.githubusercontent.com/13188580/221792818-2f46420c-9f57-442a-b8a3-23850cc44a21.png)

**Reply by Krakonos** [reply]
It's not supposed to be a grid per se, but rather indication of what parts of the map are downloaded from upstream server. So yes, it only works for downloaded regions - useful in areas where there's not much coverage to know if nothing's there, or you don't know.

If working offline, I'd suggest turning the option off, it doesn't add anything. I frequently toggle to option off once I start editing the area.

As for display safety, modern displays won't have too many issues with that I guess. Making it move with the map would actually be wanted in this regard, but not really necessary and thus very low priority.

**Reply by ghost** [reply]
Thank you for explaining!
